app-frontend/src/app/components/pages/project/layer/scenes/browse/index.html

```diff
@@ -1 +1,138 @@
+  <div class="list-group-item">
+    <i class="icon-load animate-spin" ng-class="{'stop': !$ctrl.fetchingScenes}"
+       ng-show="$ctrl.fetchingScenes"></i>
+    <strong class="color-dark">
+      Loading scenes...
+    </strong>
+  </div>
+</div>
+<div class="list-group"
+     ng-if="$ctrl.sceneList &&
+            $ctrl.sceneList.length === 0 &&
+            !$ctrl.fetchingScenes &&
+            !$ctrl.fetchError">
+  <div class="list-group-item">
+    <strong class="color-dark">
+      No scenes match this filter
+    </strong>
+  </div>
+</div>
+<div class="sidebar-scrollable list-group">
+  <rf-project-scene-item
+      ng-repeat="scene in $ctrl.sceneList track by $index"
+      scene="scene"
+      selected="$ctrl.isSelected(scene)"
+      on-select="$ctrl.onSelect(scene)"
+      processing="$ctrl.scenesBeingAdded.has(scene.id)"
+      is-previewable="true"
+  >
+    <div class="btn btn-text btn-transparent"
+         ng-repeat="action in $ctrl.sceneActions.get(scene.id) | filter: {menu: false}"
+         ng-title="{{action.title}}"
+         ng-click="action.callback()"
+         tooltips
+         tooltip-template="{{action.tooltip}}"
+         tooltip-size="small"
+         tooltip-class="rf-tooltip"
+         tooltip-side="left"
+         tooltip-hidden="{{!action.tooltip}}"
+    >
+      <i ng-class="action.icon" ng-if="action.icon"></i>
+      <i ng-class="icon.icon"
+         ng-if="icon.isActive()"
+         ng-repeat="icon in action.icons"></i>
+    </div>
+    <button class="btn btn-text btn-transparent"
+            uib-dropdown
+            ng-show="($ctrl.sceneActions.get(scene.id) | filter: {menu: true}).length > 0"
+            uib-dropdown-toggle
+    >
+      <i class="icon-menu"></i>
+      <ul class="dropdown-menu dropdown-menu-light drop-left" uib-dropdown-menu role="menu">
+        <li role="menuitem"
+            ng-repeat="action in $ctrl.sceneActions.get(scene.id) | filter: {menu: true}">
+          <a href ng-click="action.callback()"
+             ng-show="action.name"
+             ng-attr-title="{{action.title}}">{{action.name}}</a>
+          <span class="menu-separator" ng-show="action.separator"></span>
+        </li>
+      </ul>
+    </button>
+   </rf-project-scene-item>
+   <div class="sidebar-content">
+     <button class="btn btn-block btn-secondary"
+             ng-show="$ctrl.sceneList.length"
+             ng-disabled="!$ctrl.hasNext || $ctrl.fetchingScenes"
+             ng-click="$ctrl.fetchNextScenes()">
+       Load More Scenes
+       <i class="icon-load animate-spin" ng-class="{'stop': !$ctrl.fetchingScenes}"
+          ng-show="$ctrl.fetchingScenes"></i>
+     </button>
+   </div>
+</div>
+<div class="sidebar project-side-modal"
+     ng-show="$ctrl.showFilterPane"
+     style="z-index: 1001"
+>
+  <rf-scene-filter-pane
+      data-opened="$ctrl.showFilterPane"
+      data-repositories="$ctrl.repositories"
+      on-repository-change="$ctrl.onRepositoryChange(fetchScenes, repository)"
```

Comment:
The shape filter in the filter pane seems to be working well when choosing from existing shapes, but doesn't when creating a new shape or importing a shape. Additionally, the color of the text is white in shape filter input box when a shape is selected and applied.

I think this might be out of scope to fix in this PR, we could create another card for it. Plus, I think we should show where the shape is on the map.

Reply:
so the reason I didn't mess with the shape filter is because it isn't actually in the designs and I wanted to avoid doing more work in this PR.
Removing it and making other changes to the filter panel are in this issue: #4622